wrapper.go

```diff
@@ -166,11 +167,15 @@ func (hw *handlerWrapper) coldStartsDatapoint() *datapoint.Datapoint {
 }
 
 func (hw *handlerWrapper) durationDatapoint(elapsed time.Duration) *datapoint.Datapoint {
-	dp := datapoint.Datapoint{Metric: "function.duration", Value: datapoint.NewFloatValue(elapsed.Seconds()), MetricType: datapoint.Gauge}
+	dp := datapoint.Datapoint{Metric: "function.duration", Value: datapoint.NewIntValue(Milliseconds(elapsed)), MetricType: datapoint.Gauge}
 	return &dp
 }
 
 func (hw *handlerWrapper) errorsDatapoint() *datapoint.Datapoint {
 	dp := datapoint.Datapoint{Metric: "function.errors", Value: datapoint.NewIntValue(1), MetricType: datapoint.Counter}
 	return &dp
 }
+
+// Milliseconds returns the duration as an integer millisecond count.
+// Added to time.Duration in go 1.13
+func Milliseconds(d time.Duration) int64 { return int64(d) / 1e6 }
```

Comment:
Isn't this basically a reimplementation of https://golang.org/pkg/time/#Duration.Milliseconds?

Reply:
Yes but it was only added in go 1.13 and lambda supports 1.x